drivers/rwbuffer.c

```diff
@@ -839,11 +839,11 @@ int rwb_initialize(FAR struct rwbuffer_s *rwb)
       rwb->wrbuffer = kmm_malloc(allocsize);
       if (!rwb->wrbuffer)
         {
-          ferr("Write buffer kmm_malloc(%d) failed\n", allocsize);
+          ferr("Write buffer kmm_malloc(%"PRId32") failed\n", allocsize);
```

Comment:
```suggestion
          ferr("Write buffer kmm_malloc(%" PRId32 ") failed\n", allocsize);
```
nit: the string operands of the concatenation are usually separated by whitespaces.

Reply:
Done.